tailwind.config.js

```diff
@@ -50,13 +53,55 @@ export default {
           '0%': { opacity: '1' },
           '100%': { opacity: '0' },
         },
+        walletContainerIn: {
+          '0%': { opacity: '0', transform: 'translateY(-10px)' },
+          '100%': { opacity: '1', transform: 'translateY(0)' },
+        },
+        walletContainerOut: {
+          '0%': { opacity: '1', transform: 'translateY(0)' },
+          '100%': { opacity: '0', transform: 'translateY(-10px)' },
+        },
+        walletIslandContainerItemIn: {
+          '0%': { opacity: '0', transform: 'translateY(10px)' },
+          '100%': { opacity: '1', transform: 'translateY(0)' },
+        },
+        slideInFromLeft: {
+          '0%': { opacity: '0', transform: 'translateX(-20px)' },
+          '100%': { opacity: '1', transform: 'translateX(0)' },
+        },
+        slideOutToLeft: {
+          '0%': { opacity: '1', transform: 'translateX(0)' },
+          '100%': { opacity: '0', transform: 'translateX(-20px)' },
+        },
+        slideInFromRight: {
+          '0%': { opacity: '0', transform: 'translateX(20px)' },
+          '100%': { opacity: '1', transform: 'translateX(0)' },
+        },
+        slideOutToRight: {
+          '0%': { opacity: '1', transform: 'translateX(0)' },
+          '100%': { opacity: '0', transform: 'translateX(20px)' },
+        },
       },
       animation: {
         enterRight: 'fadeInRight 500ms ease-out',
         enterUp: 'fadeInUp 500ms ease-out',
         enterDown: 'fadeInDown 500ms ease-out',
         fadeIn: 'fadeIn 100ms ease-out',
         fadeOut: 'fadeOut 100ms ease-in',
+        walletContainerIn: 'walletContainerIn 300ms ease-out',
+        walletContainerOut: 'walletContainerOut 300ms forwards',
+        walletIslandContainerItem1:
+          'walletIslandContainerItemIn 300ms cubic-bezier(0.34, 1.56, 0.64, 1) 100ms forwards',
+        walletIslandContainerItem2:
+          'walletIslandContainerItemIn 300ms cubic-bezier(0.34, 1.56, 0.64, 1) 130ms forwards',
+        walletIslandContainerItem3:
+          'walletIslandContainerItemIn 300ms cubic-bezier(0.34, 1.56, 0.64, 1) 160ms forwards',
+        walletIslandContainerItem4:
+          'walletIslandContainerItemIn 300ms cubic-bezier(0.34, 1.56, 0.64, 1) 190ms forwards',
```

Comment:
Since they are specific to Wallet and WalletIsland, should these configs live elsewhere?

Reply:
see comment above, i think we should just make the names more generic as they are valid animations for any container / container element